crates/bevy_animation/src/lib.rs

```diff
@@ -646,6 +646,7 @@ fn apply_animation(
             let Ok(mut transform) = (unsafe { transforms.get_unchecked(target) }) else {
                 continue;
             };
+            #[allow(clippy::undocumented_unsafe_blocks)]
```

Comment:
I'd like to fix this one in this PR.

Reply:
I've replaced the various `#[allow(...)]` with safety comments referencing the safety comments above them and reiterating in brief